.github/workflows/create-release.yml

```
@@ -0,0 +1,45 @@
+          VERSION="${{ steps.get_version.outputs.version }}"
+          if git tag -l | grep -q "^${VERSION}$"; then
+            echo "exists=true" >> $GITHUB_OUTPUT
+            echo "Tag ${VERSION} already exists, skipping release creation"
+          else
+            echo "exists=false" >> $GITHUB_OUTPUT
+            echo "Tag ${VERSION} does not exist, will create release"
+          fi
+
+      - name: Create Release
+        if: steps.check_tag.outputs.exists == 'false'
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.10)

[error] the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)